### ISSUE TITLE
some general questions

### ISSUE THREAD

**Issue by moeinp70** [charlesq34/pointnet#70]
Hi. I've read your paper (point net) and I really appreciate your work. I have some general questions about your work as follow :
1. Is  the reported test accuracy in the paper highest accuracy  achieved? Or its calculated using other approaches, maybe averaging test accuracy over multiple times? . The way that test accuracy is calculated could be really useful for me.

2. Is the  value of num_votes always equal to one in the evaluation script?

3.What are the criteria for saving the trained model ? Under which criteria is the "log/model.ckpt" saved?

Thanks

**Comment by charlesq34** [comment]
Hi @moeinp70 Thank you for your interest!

For the first two questions please see my answers here: https://github.com/charlesq34/pointnet/issues/49

As for 3: I'm not fully sure what you means but how the model is saved is specified by Tensorflow which helps save both the parameters of the trained network as well as training dynamics like iteration number, learning rate etc.

Hope it helps.
Best,
Charles
